modules/dev2plus/hw_01/src/index.js

```diff
@@ -0,0 +1,73 @@
+const dataSet = require("./assets/dataSet");
+const { BinaryTree, sortBySalary, sortByName } = require("./Tree/BinaryTree");
+const utils = require("./utils/utils");
+
+const BinarySTree = new BinaryTree();
+
+BinarySTree.insert(dataSet);
+
+//console.log(BinarySTree.getTree());
+utils.timeMeasure("start", "inOrder");
```

Comment:
`timeMeasure` - really nice and clean solution

Reply:
Thx, but it is an initial version...I would like to fix it too :)